fhttp/httprunner.go

```diff
@@ -24,6 +24,7 @@ import (
 	"fortio.org/fortio/log"
 	"fortio.org/fortio/periodic"
 	"fortio.org/fortio/stats"
+	"golang.org/x/sync/errgroup"
```

Comment:
I prefer not adding more dependencies unless terribly valuable;
what does errgroup does that can't be done with sync.WaitGroup ?

Reply:
Not that much  - it is ultimately about 20 lines of code wrapping it to give error handling. I just inlined it instead now